README.md

```diff
@@ -61,6 +63,83 @@ ensemble_models_dir = '/model_folder'
 # e.g. '/model_folder/resnet_50/model.h5', '/model_folder/resnet_50/model_spec.json', '/model_folder/densenet201/model.h5', '/model_folder/densenet201/model_spec.json'
 
 ```
+**To evaluate on coarse classes after training on granular classes**
+
+Given a model trained on M classes and test set based on N classes (M > N), allow the evaluation on sets of classes by providing a *concept dictionary*.
+
+**Case 1: Regular evaluation**
+
+E.g. 
+Training scenario:
+```
+[class_0]
+[class_1]
+[class_2]
+[class_3]
+```
+Regular evaluation scenario:
+```
+[class_0]
+[class_1]
+[class_2]
+[class_3]
+```
+Results for regular evaluation:
+![Confusion_matrix](https://github.com/triagemd/keras-eval/blob/combine_probs/figs/confusion_matrix_granular.png)
```

Comment:
We'll need to rename `combine_probs` to `master`, otherwise the image won't be shown in the Master branch.